backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -91,7 +91,8 @@ private static IQueryable<Entity.Views.Property> GenerateCommonQuery(this PimsCo
                 }
                 if (filterAgencies.Any())
                 {
-                    query = query.Where(p => filterAgencies.AsEnumerable().Contains(p.AgencyId));
+                    var agencies = filterAgencies.Concat(context.Agencies.AsNoTracking().Where(a => filterAgencies.Contains(a.Id)).SelectMany(a => a.Children.Select(ac => (int?)ac.Id)).ToArray()).Distinct();
+                    query = query.Where(p => agencies.Contains(p.AgencyId));
```

Comment:
this fixes the issue with the "subagency" properties not being displayed when searching for properties using the "parent" agency.